server/src/main/java/org/cloudfoundry/identity/uaa/scim/bootstrap/ScimUserBootstrap.java

```diff
@@ -200,15 +200,21 @@ public void onApplicationEvent(ApplicationEvent event) {
         }
     }
 
+    private void updateScimUser(UaaUser uaaUser) {
+        ScimUser user = getScimUser(uaaUser);
+        if (user != null) {
+            updateUser(user, uaaUser, false);
+        }
```

Comment:
I am not sure if silent update failure is a good idea.  There is no indication besides a debug message that user wasn't updated.  When delete fails, it throws RuntimeException.  Maybe we should do the same for update.